ui-v2/app/components/consul-instance-checks/index.hbs

```diff
@@ -0,0 +1,34 @@
+{{#if (gt items.length 0)}}
+{{#let (service/instance-checks items) as |itemCheck| }}
```

Comment:
If you don't see us using it anywhere else, and you wanted to, you could potentially move this helper code into the component, completely up to you tho.

Reply:
I don't know how to add all that logic to the component. I'll leave it as it is for now.